package.json

```diff
@@ -6,7 +6,9 @@
   "main": "index.js",
   "scripts": {
     "prepack": "npmignore --auto --commentLines=autogenerated",
-    "lint": "eslint .",
+    "lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
+    "lint:docs": "markdownlint \"**/*.md\"",
+    "lint:js": "eslint .",
```

Comment:
```suggestion
    "prelint": "npm run lint:docs",
    "lint:docs": "markdownlint \"**/*.md\"",
    "lint": "eslint .",
```
i'm not a fan of `npm-run-all`, and i'm very very skeptical of the reliability of parallel/concurrent runners in terms of exit codes.

Reply:
Updated. Although I think we will run into the same problem again when we want to add the next type of linting. We could also use [npm-run-all](https://www.npmjs.com/package/npm-run-all) without concurrency. Note that `npm-run-all` has been stable for years and it's heavily used throughout the ecosystem.

Reply:
I've used it, as well as parallelshell and concurrently, and with all of them I've seen false positives - where one of the tasks failed but the overarching command didn't.

We can run them all in serial, manually, it'll be fine.